Sources/GRPCCore/Metadata.swift

```diff
@@ -130,6 +130,13 @@ public struct Metadata: Sendable, Hashable {
     self.elements = []
   }
 
+  /// Initialize Metadata from a Sequence of Elements
```

Comment:
Nit: add the appropriate backticks here and end with a full-stop

```suggestion
  /// Initialize `Metadata` from a `Sequence` of `Element`s.
```